src/worldcereal/openeo/feature_extractor.py

```diff
@@ -62,6 +63,25 @@ def output_labels(self) -> list:
         of the presto embeddings"""
         return [f"presto_ft_{i}" for i in range(128)]
 
+    def _compute_slope(self, inarr: xr.DataArray) -> xr.DataArray:
+        """Computes the slope using the richdem library. The input array should
+        have the following bands: 'elevation' And no time dimension. Returns a
+        new DataArray containing the new `slope` band.
+        """
+        from richdem import (  # pylint: disable=import-outside-toplevel
+            TerrainAttribute,
+            rdarray,
+        )
+
+        dem = inarr.sel(bands="elevation").values
+        dem_array = rdarray(dem, no_data=65535)
+        slope = TerrainAttribute(dem_array, attrib="slope_riserun")
+        return xr.DataArray(
+            slope[None, :, :],
+            dims=("bands", "x", "y"),
+            coords={"bands": ["slope"], "x": inarr.x, "y": inarr.y},
```

Comment:
also here the name is not (yet) compatible with presto inference but this can be changed in presto code.

Reply:
How do you prefer really? I doesn't change for me if we name it here or in the presto library

Reply:
Ideally we'd immediately address this one: https://github.com/WorldCereal/worldcereal-classification/issues/55

Reply:
So let's go for presto names `altitude` and `slope` like you did. Will do the other band name remapping of #55 in separate PR, linked to a change in presto code too.